charts/libero-reviewer/values.yaml

```diff
@@ -48,7 +48,9 @@ submission:
     sftpPath: "/upload"
     sftpPort: 22
     sftpSecret: libero-reviewer--sftp
-
+    emailPrefix: ""
+    emailRecipient: "libero-reviewer@mailinator.com"
+    apiKey: libero-reviewer--meca-api-key
```

Comment:
I would suggest using mecaApiSecret instead and adding a comment specifying which key is expected inside the secrete. This way it is more obvious that one shouldn't put the API key directly as a value.

See below:
```
  # Name of an existing secret with a 'token' entry
  elifeApiSecret: "libero-reviewer--elife-api"
```

A useful default value for `emailPrefix` would be great.

Maybe a comment regarding the purpose of the emailPrefix and emailRecipient, i.e. what kind of values are needed here for prod?